src/libcollections/str.rs

```diff
@@ -789,27 +802,37 @@ impl str {
         core_str::StrExt::split_at(self, mid)
     }
 
-    /// An iterator over the codepoints of `self`.
+    /// An iterator over the Unicode scalar values of `self`.
+    ///
+    /// In Unicode a single letter can be composed of multiple `char`s,
+    /// and a single `char` can represent multiple letters.
```

Comment:
I thought I had a clue, but I can't really follow this about multiple letters in a char. I suppose that's correct with the ligatures. However "letter" is another term thrown into the salad!

Human readable characters is the term used below and I think it is better? Also, I suppose it should say code point here too, not USV, if we go with that.